test/graph.test.ts

```diff
@@ -78,3 +79,19 @@ describe.skip.each(ages)("Test the current state of the subgraph", (age) => {
     });
   });
 });
+
+describe("Subgraph versioning", () => {
+  let usersBalances: UserBalances[];
+
+  beforeAll(async () => {
+    usersBalances = await fetchUsers(SUBGRAPH_URL, ages[2].epochs[0].finalBlock!);
+  });
+
+  it("Should distribute only from the version 1 of the script before rewards mechanism version 2", () =>
```

Comment:
This test will fail as soon as we get over the v2 timestamp, correct?

Reply:
The test is not going to fail since we are not testing for timestamp greater than V2 timestamp but the assumption is going to be false right

Ive tested stg for future epochs distribution in the next PR